examples/design-system/packages/ui/package.json

```diff
@@ -12,17 +12,18 @@
   "scripts": {
     "build": "tsup src/index.tsx --format esm,cjs --dts --external react",
     "dev": "tsup src/index.tsx --format esm,cjs --watch --dts --external react",
-    "lint": "eslint src/",
+    "lint": "eslint .",
```

Comment:
In another PR (with-npm update), there was also `--max-warnings=0`. Do we want to be consistent about that?